provisioning/plugins/corim-nitro-decoder/extractor.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/services/proto"
+	structpb "google.golang.org/protobuf/types/known/structpb"
+)
+
+type Extractor struct{}
+
+func (o Extractor) SwCompExtractor(rv comid.ReferenceValue) ([]*proto.Endorsement, error) {
+	return nil, fmt.Errorf("Not implemented, not needed?")
```

Comment:
this is where you'd read the expected PCRs from the CoMID, so it looks like it's needed

Reply:
I think needed for some use-cases, but not all (see my above comments about Veracruz's needs).

Reply:
👍 this needs designing